py/aou_workbench_client/swagger_client/models/operator.py

```diff
@@ -33,6 +33,7 @@ class Operator(object):
     GREATER_THAN_OR_EQUAL_TO = "GREATER_THAN_OR_EQUAL_TO"
     LIKE = "LIKE"
     IN = "IN"
+    BETWEEN = "BETWEEN"
```

Comment:
Note: I think this got added for another purpose, but isn't supported in our cohort materialization API right now... I will work on removing it from here.

Reply:
I added a hack to our regen script for removing this, and filed

https://precisionmedicineinitiative.atlassian.net/browse/RW-761

to track handling it or removing it properly.